afids_utils/tests/test_transforms.py

```diff
@@ -0,0 +1,31 @@
+from __future__ import annotations
+
+import numpy as np
+from hypothesis import HealthCheck, given, settings
+from numpy.testing import assert_array_equal
+from numpy.typing import NDArray
+
+from afids_utils.tests.strategies import affine_xfm, world_coord
+from afids_utils.transforms import afid_world2voxel
+
+
+class TestAfidWorld2Voxel:
+    @given(world_coord=world_coord(), nii_affine=affine_xfm())
+    @settings(
+        deadline=400,
+        suppress_health_check=[HealthCheck.function_scoped_fixture],
+    )
+    def test_transform_output(
+        self, world_coord: NDArray[np.float_], nii_affine: NDArray[np.float_]
+    ):
+        expected_voxel_coord = world_coord.T - nii_affine[:3, 3:4]
```

Comment:
This appears just to be a reimplementation of the code we're testing, which I think is an antipattern. For now why don't we just test the type and shape of the output, then once we include voxel-to-world transforms, we can test the correctness with a round-trip conversion.